PIL/Image.py

```diff
@@ -695,9 +698,9 @@ def __getstate__(self):
             self.tobytes()]
 
     def __setstate__(self, state):
-        # type: (List) -> None
+        # type: (Tuple) -> None
         Image.__init__(self)
-        self.tile = []
+        self.tile = []  # type: List[Tuple] ## FIXME More detail? Should tile be in __init__ & _new?
```

Comment:
Tile is a messy type, and really would be good to document. `ImageFile` is probably the core of where that happens, along with all of the decoders.